nni/tools/nnictl/launcher.py

```diff
@@ -624,26 +618,25 @@ def create_experiment(args):
 def manage_stopped_experiment(args, mode):
     '''view a stopped experiment'''
     update_experiment()
-    experiment_config = Experiments()
-    experiment_dict = experiment_config.get_all_experiments()
+    experiments_config = Experiments()
```

Comment:
I personally suggest rename "config" to "metadata" or "metainfo".
Not mandatory if you think "config" is better.